spec/migrations/20170120164058_create_task_for_each_job_and_transfer_attributes_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_migration
+
+describe CreateTaskForEachJobAndTransferAttributes do
+  let(:miq_tasks_stub) { migration_stub(:MiqTask) }
+  let(:jobs_stub) { migration_stub(:Job) }
+
+  migration_context :up do
+    it "creates associated task for each job and assigns to task the same name" do
+      jobs_stub.create!(:name => "Hello Test Job", :status => "Some test status", :miq_task_id => nil)
+      jobs_stub.create!(:name => "Hello Test Job2", :state => "Some state", :miq_task_id => nil)
```

Comment:
Before this PR, can jobs already link to a task?

If so, test with a job already linked to a `MiqTask`.